backend/isoko-backend/__tests__/unit/handlers/review/post-review.test.js

```diff
@@ -0,0 +1,190 @@
+const {
+   postReviewHandler,
+} = require('../../../../src/handlers/review/post-review.js');
+const dynamodb = require('aws-sdk/clients/dynamodb');
+const { BUSINESS_TABLE } = require('../../../../src/constants');
+
+jest.mock('aws-sdk/clients/dynamodb');
+
+describe('PostReviewHandler tests', () => {
+   let putSpy;
+
+   beforeAll(() => {
+      putSpy = jest.spyOn(dynamodb.DocumentClient.prototype, 'put');
+   });
+
+   afterAll(() => {
+      putSpy.mockRestore();
+   });
+
```

Comment:
after the afterAll() function, add this: 

afterEach(() => {
        putSpy.mockReset();
     });

makes sure you're starting with a "fresh" mock every time you run a test so you won't run into the weird error messages i got

Reply:
yup added!